core/src/components/input/input.ionic.scss

```diff
@@ -1,26 +1,127 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
 @use "./input.common";
-@forward "./input.ionic.outline.scss";
-
+@forward "./input.ionic.solid.scss";
 // Ionic Input
 // --------------------------------------------------
 
 :host {
   --color: #{globals.$ion-primitives-neutral-1200};
   --border-width: #{globals.$ion-border-size-025};
   --border-color: #{globals.$ion-primitives-neutral-500};
+  --border-radius: #{globals.$ion-border-radius-100};
```

Comment:
Where is this value coming from? In Figma I see the following:

`round` -> `round.xl` -> `16px` -> `$ion-round-xl` (`$ion-border-radius-400` -> `$ion-scale-400: var(--ion-scale-400, 16px);`)
`soft` -> `soft.xl` -> `8px` -> `$ion-soft-xl` (`$ion-border-radius-200` -> `$ion-scale-200: var(--ion-scale-200, 8px);`)
`rectangular` -> `rectangular.xl` -> `0px` -> `$ion-rectangular-xl` (`$ion-border-radius-0` -> `$ion-scale-0: var(--ion-scale-0, 0px);`)

Reply:
It was from [here](https://github.com/ionic-team/ionic-framework/pull/30417/files#diff-cbf91c9b97e425d969df4730e12b3424dd54de7664b9e629d455d2e63539c84eL7), I can delete it, but it's possible to have an undefined shape when in ionic theme [here](https://github.com/ionic-team/ionic-framework/pull/30417/files#diff-a39147111a459c7d50e285f51eddf2d3fee983bda67654c842a2c645565af6bcR566)

Reply:
The `undefined` shape is only when it isn't the `ionic` theme:

![Screenshot 2025-05-22 at 2 04 01 PM](https://github.com/user-attachments/assets/1fd6f994-1150-4b02-a2e2-88d64cb70fa4)